alfresco-actuators/src/main/java/eu/xenit/actuators/Health.java

```diff
@@ -1,14 +1,18 @@
 package eu.xenit.actuators;
 
-import java.util.Map;
-
 public class Health {
 
-    public static final String KEY_OUTPUT = "output";
-    public static final String KEY_ERROR = "error";
+    private HealthStatus status;
+    private Object details;
```

Comment:
@JanVdloock Why do you use `Object` and not something more specific?

Reply:
To allow all possible type of details in the health information. If some new info is added, everything will continue to work correctly, since the ObjectMapper takes care of the JSON formatting.